go/vt/vtexplain/vtexplain_vttablet.go

```diff
@@ -494,16 +494,44 @@ func (t *explainTablet) HandleQuery(c *mysql.Conn, query string, callback func(*
 	// return the pre-computed results for any schema introspection queries
 	tEnv := t.vte.getGlobalTabletEnv()
 	result := tEnv.getResult(query)
-	emptyResult := &sqltypes.Result{}
-	if sidecardb.MatchesVTInitQuery(query) {
+	//emptyResult := &sqltypes.Result{}
+	fmt.Printf("query: %s\n", query)
+	/*if sidecardb.MatchesVTInitQuery(query) {
 		return callback(emptyResult)
 	}
-	if strings.HasPrefix(query, "use _vt") || strings.HasPrefix(query, "CREATE TABLE _vt") || strings.HasPrefix(query, "ALTER TABLE _vt") || strings.HasPrefix(query, "CREATE TABLE IF NOT EXISTS _vt") {
+	if //strings.HasPrefix(query, "use `fakesqldb`") ||
+	strings.HasPrefix(query, "use _vt") ||
+		strings.HasPrefix(query, "CREATE TABLE _vt") ||
+		strings.HasPrefix(query, "ALTER TABLE _vt") ||
+		strings.HasPrefix(query, "CREATE TABLE IF NOT EXISTS _vt") {
 		return callback(emptyResult)
-	}
+	}*/
 	if result != nil {
 		return callback(result)
 	}
+
+	if err := t.db.GetRejectedQueryResult(query); err != nil {
+		return err
+	}
```

Comment:
Let me give you some context. FakeSqlDB is used in unit test and vtexplain. Yes there are different data structure define to where we can declare statement and there expected result. go/mysql/fakesqldb/server.go has handleQuery which uses these structure but in vtexplain which also implement QueryHandler interface does not use these structure. instead of having this HasPrefix checks, I made the code consistent with how QueryHandler is implemented in fakesqldb/server.go.